Buildings/Controls/OBC/CDL/Discrete/DiscreteMovingMean.mo

```diff
@@ -0,0 +1,104 @@
+within Buildings.Controls.OBC.CDL.Discrete;
+block DiscreteMovingMean "Discrete moving mean of a sampled input signal"
+  parameter Integer nSam(min=2) "Number of samples to be averaged";
```

Comment:
I prefer to simply use `n` (which is shorter and customary for the size of samples).